deploy/webhooks.yaml

```diff
@@ -139,5 +139,6 @@ webhooks:
     - UPDATE
     resources:
     - ssps
+    scope: Cluster
```

Comment:
The SSP CRD isn't cluster-scoped, see here: https://github.com/kubevirt/hyperconverged-cluster-operator/blob/20df7b1db67c06d4d7edc3fe362d2be8e09e27fe/deploy/crds/scheduling-scale-performance00.crd.yaml#L15

Reply:
reverted